databases/redis-enterprise-operator/manifests/06-performance-and-ha/ha-cluster-role.yaml

```diff
@@ -12,6 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 ---
+# [START gke_manifests_ha_redis_enterprise_cluster_role]
```

Comment:
Looks great, but would you mind prefixing them with `gke_databases_redis_enterprise_operator_`?

So for example this could be: `gke_databases_redis_enterprise_operator_ha_cluster_role`

There's a lot of tags that exist, I just wanna make sure they're namespaced properly so they're easier to find